wallet/src/wallet/tests.rs

```diff
@@ -57,10 +57,34 @@ fn gen_random_password(rng: &mut (impl Rng + CryptoRng)) -> String {
     (0..rng.gen_range(1..100)).map(|_| rng.gen::<char>()).collect()
 }
 
+fn gen_random_transfer(rng: &mut (impl Rng + CryptoRng), amount: Amount) -> TxOutput {
+    let destination = Destination::PublicKey(
+        crypto::key::PrivateKey::new_from_rng(rng, crypto::key::KeyKind::Secp256k1Schnorr).1,
+    );
+    match rng.gen::<bool>() {
+        true => TxOutput::Transfer(OutputValue::Coin(amount), destination),
+        false => {
+            let lock_for_blocks = rng.gen::<u32>() as u64;
```

Comment:
sorry this will probably overflow BlockDistance. But I think other locals can be generated as u64

Reply:
replaced with gen_range, as even with u32 it can overflow if both seconds_between_blocks and lock_for blocks get u32::max and then adding the result to the current_timestamp